.github/workflows/pitest.yml

```diff
@@ -35,6 +48,19 @@ jobs:
   pitest-ant:
     runs-on: ubuntu-latest
     steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Install Groovy 4.0.3
+        run: |
+          GROOVY_LINK="https://groovy.jfrog.io/artifactory/dist-release-local/groovy-zips/apache-groovy-binary-4.0.3.zip"
+          wget -O ${{ runner.temp }}/groovy.zip $GROOVY_LINK;
+          unzip ${{ runner.temp }}/groovy.zip -d ${{ runner.temp }};
+          mv ${{ runner.temp }}/groovy-4.0.3 ${{ runner.temp }}/groovy
+          GROOVY_HOME=${{ runner.temp }}/groovy
+          echo "GROOVY_HOME=${{ runner.temp }}/groovy" >> $GITHUB_ENV
+          echo "$GROOVY_HOME/bin" >> $GITHUB_PATH
```

Comment:
This will be extracted to a script once we agree upon the version of groovy to use. Same for the others.

Reply:
resolved, as we are using sdkman instead.